doc/man/man3/crypto_x25519.3monocypher

```diff
@@ -1 +0,0 @@
-crypto_key_exchange.3monocypher
```

Comment:
I only just noticed this field, and I haven't been updating it with my other tweaks

is that a problem?

Reply:
Oh yeah, thanks for the reminder. It's not a problem. I was planing to dedicate a commit to those dates, to mark the day the manual was last reviewed.